.github/workflows/stale.yml

```diff
@@ -0,0 +1,28 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '0 13 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v4
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is GITHUB_TOKEN added into secrets?

Reply:
This was the value in the yaml when I went to create a new workflow in the github UI so I kept it. Not sure if there is another way to do this but I figured it was a variable defined by github and not by us.

Reply:
It looks like this will automatically be created when we enable this workflow.

> GitHub automatically creates a GITHUB_TOKEN secret to use in your workflow. You can use the GITHUB_TOKEN to authenticate in a workflow run.

https://docs.github.com/en/actions/reference/authentication-in-a-workflow